app/constants.js

```diff
@@ -2,3 +2,6 @@ exports.regDomainWolox = /^(([^<>()[\]\\,;:\s@”]+((@wolox)+\.(co))))$/;
 exports.regPasswordAlphaNumeric = /^(?=.*\d)(?=.*\D)/;
 
 exports.expiresIn = 30 * 30;
+
+exports.roleAdmin = 1;
```

Comment:
I think, it would be better if you map the values ​​of the roles in an object something like,  roles: {
    REGULAR: '0',
    ADMIN: '1'
  }